src/webots/engine/WbAnimationRecorder.cpp

```diff
@@ -435,7 +435,8 @@ void WbAnimationRecorder::stopRecording() {
   // write initial state
   out << "{\"time\":0,\"poses\":[";
   if (commandsChangedFromStart.isEmpty()) {
-    WbLog::info(tr("Error: No animation content is available because the simulation did not start."));
+    WbLog::info(tr("Error: No animation content is available because no changes occurred in the simulation. "
+                   "If you just want a 3D environment file, consider exporting a scene instead"));
```

Comment:
It would be nice to add a final dot at the end of the second sentence.

Reply:
Ok, Daniel did it in commit https://github.com/cyberbotics/webots/pull/5447/commits/10b706568d9523b3bbcebf17a29d75a9c5497ce7